compile.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
This file seems to be formatted with tabs or use more than 2 spaces for indenting - can you check?